src/index.html

```diff
@@ -6,11 +6,50 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      class="form-wrapper first-wrapper"
+      data-qa="big"
+    >
+      <img
```

Comment:
Generally you should put empty lines between siblings of a parent, here the parent is form and siblings are img and input, an example of such formatting

```html
<form>
  <img>

  <input>  
</form>
```
There are some exception from this rule, for example if you have some repeated elements like in an ul, you can omit the empty lines:
```html
<ul>
  <li>Text</li>
  <li>Text</li>
  <li>Text</li>
  <li>Text</li>
</ul>
```

Reply:
Also this img here should be wrapped inside a label for the input, otherwise upon clicking on the img input is not being focued